client/src/pages/private/SiteViewDetails.js

```diff
@@ -103,6 +103,10 @@ export default ({ match }) => {
       'Phone Number': 'operatorPhone',
       'Email Address': 'operatorEmail',
     },
+    'Positions Overview': {
+      'HCAP Hires': 'hcapHires',
+      'Non-HCAP': 'nonHcapHires',
```

Comment:
This is nitpicky but I think we should be super explicit with our naming here - designs have us potentially reporting on more of the funnel later and I think it could confuse people to have a discrepancy. So I'm thinking:
- HCAP Hires
- Non-HCAP Hires

It's repetitive but I think it will help users so they aren't wondering if it's showing them something slightly different because the naming is slightly different.